e2e/wsl-integrations.e2e.spec.ts

```diff
@@ -210,83 +210,94 @@ test.describe('WSL Integrations', () => {
     expect(wslIntegrationList.getByText('alpha')).not.toBeNull();
     expect(wslIntegrationList.getByText('beta')).not.toBeNull();
     expect(wslIntegrationList.getByText('gamma')).not.toBeNull();
-  });
 
-  /*
-  test('should show checkbox states', (async() => {
-    const integrations = wslPage.wslIntegrations;
-    const alpha = integrations.find(item => item.name === 'alpha');
-    const beta = wslPage.getIntegration('beta');
-    const gamma = wslPage.getIntegration('gamma');
-
-    await expect(alpha.locator).toHaveCount(1);
-    await expect(alpha.checkbox).not.toBeChecked();
-    await expect(alpha.name).toHaveText('alpha');
-    await expect(alpha.error).not.toBeVisible();
-
-    await expect(beta.locator).toHaveCount(1);
-    await expect(beta.checkbox).toBeChecked();
-    await expect(beta.name).toHaveText('beta');
-    await expect(beta.error).not.toBeVisible();
-
-    await expect(gamma.locator).toHaveCount(1);
-    await expect(gamma.checkbox).not.toBeChecked();
-    await expect(gamma.name).toHaveText('gamma');
-    await expect(gamma.error).toHaveText('some error');
+    expect(await wslPage.alpha.isChecked()).toBeFalsy();
+    expect(await wslPage.beta.isChecked()).toBeTruthy();
+    expect(await wslPage.gamma.isChecked()).toBeFalsy();
+
+    const craftyErrorMessage = 'Error: some error';
+    let parent = wslPage.page.locator('[data-test="item-alpha-parent"]');
+
+    await expect(parent.filter({ hasText: craftyErrorMessage })).toHaveCount(0);
+    parent = wslPage.page.locator('[data-test="item-beta-parent"]');
+    await expect(parent.filter({ hasText: craftyErrorMessage })).toHaveCount(0);
+    parent = wslPage.page.locator('[data-test="item-gamma-parent"]');
+    await expect(parent.filter({ hasText: craftyErrorMessage })).toHaveCount(1);
   });
 
   test('should allow enabling integration', async() => {
+    // This is how we do a reload...
     const { wsl: wslPage } = new PreferencesPage(preferencesWindow);
-    await wslPage.reload();
-    const integrations = wslPage.integrations;
 
-    await expect(integrations).toHaveCount(1, { timeout: 10_000 });
+    await wslPage.tabIntegrations.click();
+    await expect(wslPage.wslIntegrations).toBeVisible();
 
-    const alpha = wslPage.getIntegration('alpha');
+    await expect(wslPage.wslIntegrations).toHaveCount(1, { timeout: 10_000 });
+    let alpha = wslPage.alpha;
 
-    await expect(alpha.checkbox).not.toBeChecked();
-    await alpha.assertEnabled();
-    await alpha.click();
-    await alpha.assertDisabled();
+    expect(await alpha.isChecked()).toBeFalsy();
+    expect(await alpha.isEnabled()).toBeTruthy();
+    // Don't know why force-true is necessary, playwright times out without it.
+    await alpha.click({ force: true });
```

Comment:
Note that this is forcing a click on the `<input type="checkbox">`, but normally we'd be interacting with the owning `<Checkbox>` Vue component instead… That's why we had [`WSLIntegrationsPage.getIntegration(): CheckboxLocator`](https://github.com/rancher-sandbox/rancher-desktop/blob/3ff502878c4d257a29c49ceea11c312d429d5a15/e2e/pages/wsl-integrations-page.ts#L57).